bin/init_exercise.py

```diff
@@ -0,0 +1,411 @@
+#!/usr/bin/env python
+"""
+Script to initialize an exercise for Exercism's Rust track
+
+Why a Python script in the Rust track repo? Distribution.
+A rust program would either need to be precompiled for various
+platforms, and available for download (in which case it wouldn't
+conveniently work in the repository), or would need to be included
+as a sub-crate and compiled locally. A python script can simply

```

Comment:
I would not think it would be so bad to include this code as a crate... except for the fact that I would for personal reasons have chosen a language other than Rust to write this code in. Perhaps the fact that you have written this code is also evidence that you also have similar reasons.

The only potential snag is that perhaps it may confuse contributors to this repository, but as I noted in https://github.com/exercism/rust/pull/288#issuecomment-301280919 we already have many scripts in a non-Rust language (shell scripts!), so we do not necessarily make the world any worse by adding a different non-Rust language.

Reply:
Agreed. My take on the utility of including this script is that it's a tool which helps contributors, but it is in no way required, so adding it can't make the world worse for a new contributor if they don't have the ability to run it.

And yes, when initially choosing the language, the fact that Python is (for me, for projects of this size) simply faster to code in was a strong incentive. I could write the script in Python in a day; if I'd started it in Rust, it wouldn't yet be ready.